mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageMapperTest.java

```diff
@@ -40,11 +41,21 @@
 import org.junit.After;
 import org.junit.Before;
 import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.rules.TestRule;
 
 public class CassandraMessageMapperTest extends MessageMapperTest {
     
     @ClassRule public static DockerCassandraRule cassandraServer = new DockerCassandraRule();
-    
+
+    public static ContainerLifecycleConfiguration cassandraLifecycleConfiguration = ContainerLifecycleConfiguration.builder()
+        .iterationsBetweenRestart(25)
```

Comment:
In _CassandraGetMessageListMethodTest_ it's set to 20, can we use the same value everywhere (constant)?

Reply:
What is the main reason of that? Compaction or something like that?

Reply:
Well, if you want